File: WYNKWeb/src/app/Models/ViewModels/cancel-poreg-view.model.ts




export class CancelPORegView {

  getC_PO_Reg: Array<getC_PO_Reg> = [];
}


export class getC_PO_Reg {

  PoNo: string; 
  
  PoDate: string; 
  
  //Q_No: number;
     
  //Q_Date: number;
  Parent_No: number;
  Parent_Date: number;

       
  VendorName: string; 
        
  Dlvry_Name: string;
         
  Dlvry_Addrss: string;
        
  Location: string; 
           
  Gross_Prod_val: number;
  Tot_Dis_val: number;
           
  Tot_Tax_val: number;
                      
  //Tot_CGSTTax_val: number;
 // Tot_SGSTTax_val: number;
  Tot_Po_val: number;
 
  PO_status: string;
 
  Parent_PO_No: number;

  Parent_PO_Date: number;




}

File: WYNK.Data/Model/ViewModel/CancelPORegViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WYNK.Data.Model.ViewModel
{
    public class CancelPORegViewModel
    {
        public PurchaseOrderTrans PurchaseOrderTrans { get; set; }
        public PurchaseOrder PurchaseOrder { get; set; }
        public Vendor_Masters Vendor_Masters { get; set; }
        public Drug_Master drugMaster { get; set; }
        public OneLine_Masters OneLine_Masters { get; set; }


        public ICollection<getC_PO_Reg> getC_PO_Reg { get; set; }

    }


    public class getC_PO_Reg
    {
        public string PoNo { get; set; }
        public DateTime PoDate { get; set; }
        // public string Q_No { get; set; }
        // public DateTime? Q_Date { get; set; }
        public string Parent_No { get; set; }
        public DateTime? Parent_Date { get; set; }
        public string VendorName { get; set; }
        public string Dlvry_Name { get; set; }
        public string Dlvry_Addrss { get; set; }
        public string Location { get; set; }
        public decimal Gross_Prod_val { get; set; }
        public decimal Tot_Dis_val { get; set; }
        public decimal Tot_Tax_val { get; set; }
        public decimal Tot_CGSTTax_val { get; set; }
        public decimal Tot_SGSTTax_val { get; set; }
        public decimal Tot_Po_val { get; set; }
        public string PO_status { get; set; }
        public string Parent_PO_No { get; set; }
        public DateTime Parent_PO_Date { get; set; }



    }



}
